lib/settings/pages/general_settings_page.dart

```diff
@@ -405,11 +406,12 @@ class _GeneralSettingsPageState extends State<GeneralSettingsPage> with SingleTi
                         ListOption(
                           description: LocalSettings.defaultFeedSortType.label,
                           value: ListPickerItem(label: defaultSortType.value, icon: Icons.local_fire_department_rounded, payload: defaultSortType),
-                          options: allSortTypeItems,
+                          options: [...SortPicker.getDefaultSortTypeItems(includeVersionSpecificFeature: IncludeVersionSpecificFeature.never), ...topSortTypeItems],
                           icon: Icons.sort_rounded,
                           onChanged: (_) {},
                           isBottomModalScrollControlled: true,
                           customListPicker: SortPicker(
+                            includeVersionSpecificFeature: IncludeVersionSpecificFeature.never,
```

Comment:
Just to clarify, the default feed/comment sort options in the settings page won't allow the user to select the new sort types right?

If thats the case, then I think its okay to leave it as is for now. Eventually, we would want to allow them the option to select those (and if the lemmy instance doesn't support the selected sort, then we can default to one of available sort types)

Reply:
Yes, exactly. I figured for now, most instances won't have these, so users may be confused if they select one of the new ones by default and don't see it anywhere. Once we get a feeling that many instances are upgraded, then we can add them (with a fallback, as you said).

Reply:
Sounds good!